src/style.css

```diff
@@ -1 +1,70 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  line-height: 14.06px;
+  font-weight: 500;
+  text-decoration: none;
+  color: #000;
+}
+
+.container {
+  width: 100%;
+  padding-left: 50px;
+  padding-right: 50px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.box-1 {
+  display: flex;
+  flex-direction: row;
+  padding: 22.5px 0;
+  margin-left: 20px;
+}
+
+.box-1:first-child {
+  margin-left: 0;
+  padding: 0;
+}
+
+.logo {
+  display: flex;
+  align-self: baseline;
+  margin-top: 10px;
+  cursor: pointer;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  text-transform: uppercase;
+  list-style: none;
+  display: flex;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
```

Comment:
move color to variable and use it

Reply:
I don't understand it. Where should I move color?